src/components/bottom-bar.scss

```diff
@@ -1,16 +1,49 @@
 @import "common.scss";
 
+.fireEventButton {
+  width: 98px;
+
+  .fireEventIcon {
+    width: 84px;
+    height: 46px;;
```

Comment:
Double colon - it looks like it is winking at me ;)

Reply:
Thanks. I'm gonna fix it in the first upcoming PR to avoid jumping between branches.